src/usecases/retrieveWardById.js

```diff
@@ -3,14 +3,15 @@ const retrieveWardById = ({ getDb }) => async (wardId, trustId) => {
   console.log("Retrieving ward for  ", wardId);
   try {
     const ward = await db.oneOrNone(
-      "SELECT * FROM wards WHERE id = $1 AND trust_id = $2 LIMIT 1",
+      "SELECT wards.id as ward_id, wards.name as ward_name, (SELECT name from hospitals where id=wards.hospital_id) as hospital_name, wards.hospital_id as hospital_id FROM wards WHERE wards.id = $1 AND wards.trust_id = $2 LIMIT 1",
```

Comment:
Can you add some whitespace to this please, will be easier to read and spot the subquery!